### DISCUSSION TITLE
Would love a Center Two Thirds window option

### DISCUSSION THREAD

**Discussion by totcycle** [rxhanson/Multitouch-Community#182] [Ideas]
We have right and left two thirds, and center half and thirds, but my preferred window config is often a Center Two Thirds. 

Would love to be able to assign this to trackpad and keyboard shortcuts, thanks! Would seem to fit in well with currently available options.

**Comment by rxhanson** [comment]
Good point, I thought this one was in there already. I'll add it to the backlog.

**Comment by totcycle** [comment]
Thanks Ryan, there are many many options already so that's understandable  :) 
Have been loving Multitouch and Charmstone, and also just found HazeOver seems to be playing well with your utilities.
